lib/ain-grpc/src/transaction_request.rs

```diff
@@ -54,7 +56,10 @@ impl From<TransactionRequest> for Option<TransactionMessage> {
                 gas_price: req.gas_price.unwrap_or_default(),
                 gas_limit: req.gas.unwrap_or_default(),
                 value: req.value.unwrap_or_default(),
-                input: req.data.map(Bytes::into_vec).unwrap_or_default(),
+                input: req
+                    .input
+                    .map(Bytes::into_vec)
+                    .unwrap_or(req.data.map(Bytes::into_vec).unwrap_or_default()),
```

Comment:
This should be a cleaner way to achieve this.

```suggestion
                input: req
                    .input
                    .or_else(|| req.data)
                    .map(Bytes::into_vec)
                    .unwrap_or_default(),
```